### PR TITLE
Fixes#482

### PR DESCRIPTION
**Pull request by ctaylo37**
Added popovers with descriptions.  Still need the following descriptions:

Turfgrass
Chaparral
Sand
Loam
Sandy Clay
Clay Loam

### CODE REVIEW

File: src/mmw/apps/water_balance/templates/home_page/index.html [hunk 1]
```diff
@@ -329,4 +329,16 @@ <h1 class="infil">3.0"</h1>
 {% block javascript %}
     <script type="text/javascript" src="{% static 'js/vendor_water_balance.js' %}"></script>
     <script type="text/javascript" src="{% static 'js/main_water_balance.js' %}"></script>
+    <script type="text/javascript">
+        if( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) ) {
```
Comment (lliss) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: Not thrilled about userAgent inspection for features.

File: src/mmw/apps/water_balance/templates/home_page/index.html [hunk 2]
```diff
@@ -199,84 +199,84 @@ <h1 class="infil">3.0"</h1>
 
                     <li id="thumb-lir" class="active"> <!-- LIR Thumb -->
                         <a href="#land-lir" aria-controls="filter-tab" role="tab" data-toggle="tab">
-                            <img src="{% static 'images/water_balance/thumb_lir.png' %}" alt="Low Intensity Residential">
+                            <img src="{% static 'images/water_balance/thumb_lir.png' %}" alt="Low Intensity Residential" data-container="body" data-toggle="popover" data-placement="left" data-title="Low Intensity Residential" data-content="Areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 20% to 49% percent of total cover. These areas most commonly include single-family housing units.">
```
Comment (lliss) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: We should consider moving these descriptions to a centralized place.

**Comment by chitchens** [comment]
If we want to get these last few descriptions in before merging, we have the descriptions. The soil texture descriptions are on p103 of this: http://www.cpesc.org/reference/tr55.pdf

(that's p103 of the actual PDF, not in the internal numbering scheme of the document)

Group A, B, C, and D in that document refer to Sand, Loam, Sandy Clay, and Clay Loam, respectively. Let me know if the descriptions need to be shortened to fit the design.

For the 2 remaining land cover types, Turfgrass=21, Developed Open space and Chaparral=52, Shrub/Scrub from the standard NLDC classifications: http://www.mrlc.gov/nlcd11_leg.php

**Comment by ctaylo37** [comment]
Thanks @chitchens .  I will get these in.

**Comment by chitchens** [comment]
Here's the cheat sheet:

**Sand**
These soils have low runoff potential and high infiltration rates even when thoroughly wetted. They consist chiefly of deep, well to excessively drained sand or gravel and have a high rate of water transmission (greater than 0.30 in/hr).

**Loam**
These soils have moderate infiltration rates when thoroughly wetted and consist chiefly of moderately
deep to deep, moderately well to well drained soils with moderately fine to moderately coarse textures. These soils have a moderate rate of water transmission (0.15-0.30 in/hr).

**Sandy Clay**
These soils have low infiltration rates when thoroughly wetted and consist chiefly of soils with a layer that impedes downward movement of water and soils with moderately fine to fine texture. These soils have a low rate of water transmission (0.05-0.15 in/hr).

**Clay Loam**
These soils have high runoff potential. They have very low infiltration rates when thoroughly wetted and consist chiefly of clay soils with a high swelling potential, soils with a permanent high water table, soils with a claypan or clay layer at or near the surface, and shallow soils over nearly impervious material. These soils have a very low rate of water transmission (0-0.05 in/hr).

**Turfgrass**
Areas with a mixture of some constructed materials, but mostly vegetation in the form of lawn grasses. Impervious surfaces account for less than 20% of total cover. These areas most commonly include large-lot single-family housing units, parks, golf courses, and vegetation planted in developed settings for recreation, erosion control, or aesthetic purposes. 

**Chaparral**
Areas dominated by shrubs; less than 5 meters tall with shrub canopy typically greater than 20% of total vegetation. This class includes true shrubs, young trees in an early successional stage or trees stunted from environmental conditions.

**Pull request closed without merging**